R/internal.R

```diff
@@ -5,7 +5,7 @@ log_if_verbose <- function(...) {
 }
 
 stopifnoschema <- function(table_name) {
-  assertthat::assert_that("ident" %in% class(table_name), msg = "Table name must be result of dbplyr::in_schema()")
+  assertthat::assert_that(any(c("ident","dbplyr_schema") %in% class(table_name)), msg = "Table name must be result of dbplyr::in_schema()")
```

Comment:
No need whatsoever to change this, but when there are multiple acceptable classes, `inherits()` is handy:

```
     ‘inherits’ indicates whether its first argument inherits from any
     of the classes specified in the ‘what’ argument.
```

So, this could be:

```r
assertthat::assert_that(inherits(table_name, c("ident", "dbplyr_schema")), msg = "Table name must be result of dbplyr::in_schema()")
```

Reply:
Ooh, thanks for that, I like that a lot better.

Reply:
If you've one line changes like this, please feel free to offer them directly as commits in the github interface here: ![](https://cdn.zappy.app/e3596fd0ae2fe276bfba9156472995ce.png)